### ISSUE TITLE
Add detailed Github Wiki documentation

### ISSUE THREAD

**Issue by petya-kangalova** [hotosm/fAIr#128]
Add clear documentation about fAIr in the  https://github.com/hotosm/fAIr/tree/master/docs including:

- [x] About
- [x] Home
- [x] Code of conduct
- [x] Contributor Guidelines
- [ ] FAQs

As discussed for @neelimagoogly  to work on this week. Closing this one https://github.com/hotosm/fAIr/issues/113

**Comment by omranlm** [comment]
@petya-kangalova  I think we have some FAQs now after some HUBs feedback, can we pick this up?

**Comment by petya-kangalova** [comment]
Yes! @omranlm  I was actually thinking to update this md https://github.com/hotosm/fAIr/blob/master/docs/FAQ.md. Will also get the interns to support with updates next week @edenxcodes - we can ask them to create PRs to add FAQs that I can share.

@omranlm  let's speak tomorrow as my main question is how we want to pull all these docs into fAIr frontend.

**Comment by MissTipo** [comment]
Please assign this issue to me

**Comment by kshitijrajsharma** [comment]
Go ahead ! MissTipo

**Comment by haruharu1017** [comment]
Hi kindly assign this issue to me

**Comment by MissTipo** [comment]
Hi Anastasia, I've been assigned this issue already

**Comment by KimFarida** [comment]
Hello @kshitijrajsharma may i please take on this issue with @MissTipo

**Comment by nwekealex65** [comment]
Hello @petya-kangalova you mentioned sharing FAQ which we can add to the  https://github.com/hotosm/fAIr/blob/master/docs/FAQ.md, do you mind sharing it so I can work on it?

**Comment by petya-kangalova** [comment]
@nwekealex65  thank you! This task is not specific to the Outreachy contributor period.  See document for FAQ here: https://github.com/hotosm/fAIr/issues/196

**Comment by nwekealex65** [comment]
@petya-kangalova which of the task is open for the contributor period? the raw-data-api?

**Comment by petya-kangalova** [comment]
@nwekealex65 these are the two projects. Please read all the instructions on the site:
https://www.outreachy.org/outreachy-june-2024-internship-round/communities/humanitarian-openstreetmap-team/

Here are the issues for contribution:

Create a Python-based mobile application to collect OpenStreetMap data in the field: https://github.com/hotosm/osm-fieldwork/issues/204

Create backend tests and API documentations fAIr : (https://github.com/hotosm/fAIr/issues/229) and Raw Data API (https://github.com/hotosm/raw-data-api/issues/219)

**Comment by spwoodcock** [comment]
> @nwekealex65 these are the two projects. Please read all the instructions on the site:
> https://www.outreachy.org/outreachy-june-2024-internship-round/communities/humanitarian-openstreetmap-team/
> 
> Here are the issues for contribution:
> 
> Create a Python-based mobile application to collect OpenStreetMap data in the field: https://github.com/hotosm/osm-fieldwork/issues/43
> 
> Create backend tests and API documentations fAIr : (https://github.com/hotosm/fAIr/issues/229) and Raw Data API (https://github.com/hotosm/raw-data-api/issues/219)

It's issue https://github.com/hotosm/osm-fieldwork/issues/204 for osm-fieldwork, as 43 was the old task that was already complete 👍

**Comment by petya-kangalova** [comment]
Thanks @spwoodcock  fixed! Apologies

**Comment by princyym** [comment]
Hello can i work on this can you assign me @petya-kangalova
